app/javascript/articles/components/ContentTitle.jsx

```diff
@@ -1,43 +1,38 @@
 import { h } from 'preact';
-import PropTypes from 'prop-types';
-
 import { articlePropTypes } from '../../src/components/common-prop-types';
 
-export const ContentTitle = ({ article, currentTag }) => (
-  <h3>
-    {article.flare_tag && currentTag !== article.flare_tag.name && (
-      <span
-        className="tag-identifier"
-        style={{
-          background: article.flare_tag.bg_color_hex,
-          color: article.flare_tag.text_color_hex,
-        }}
-      >
-        {`#${article.flare_tag.name}`}
```

Comment:
The flare tag has been removed. I imagine it does not fit with the new design, but just wondering if this is something we still want to keep @benhalpern?

Before:

![image](https://user-images.githubusercontent.com/833231/78928692-132e9580-7a6f-11ea-835c-c3d89efc3eb8.png)

After:

![image](https://user-images.githubusercontent.com/833231/78928725-1de92a80-7a6f-11ea-81eb-d42334436846.png)

Reply:
This is what I’ve been thinking about instead: https://github.com/thepracticaldev/dev.to/pull/7151#discussion_r405443427